phpunit.xml.dist

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="vendor/autoload.php" backupGlobals="true" colors="true" >
+  <php>
+    <ini name="error_reporting" value="32767"/>
+    <var name="namespaces" value=""/>
```

Comment:
Remove this line about namespaces, it's not correct. Throws an exception expecting an array.